components/elm/src/cpl/lnd_import_export.F90

```diff
@@ -1489,6 +1511,14 @@ subroutine lnd_export( bounds, lnd2atm_vars, lnd2glc_vars, l2x)
           end do
        end if
 
+       ! iac coupling - need an iac_active logical somewhere
```

Comment:
Confusing comment

Reply:
@aldivi : If I am understanding it correctly, we should remove this comment.

Reply:
Removed the comment.